docs/sources/features/datasources/postgres.md

```diff
@@ -218,9 +226,9 @@ tags | Optional field name to use for event tags as a comma separated string.
 Time series queries should work in alerting conditions. Table formatted queries is not yet supported in alert rule
 conditions.
 
-## Configure datasource with provisioning
+## Configure the Datasource with Provisioning
```

Comment:
If you update this for postgres you should do the same update for all datasources